aiounifi/controller.py

```diff
@@ -61,8 +61,18 @@ def __init__(
         self.devices = None
         self.wlans = None
 
+        self.headers = None
+
     async def check_unifi_os(self):
         response = await self.request("get", include_site=False)
+
+        headers = {
```

Comment:
These headers are set in my browser by response headers in the original login request if I don't already have a cookie, or if my cookie has expired. Both of these headers have to be set in order for the UDM Pro to accept the request. If either of these headers are missing, I would get the same 404 you were experiencing.

I don't know much about the `aiohttp.CookieJar`, but I feel like these values should have been included in that.

Reply:
Thanks, this did the trick